docs/bcd-workflow.md

```diff
@@ -0,0 +1,34 @@
+# BCD workflow
+
+The [web-platform-dx/web-features](https://github.com/web-platform-dx/web-features/) and the [mdn/browser-compat-data](https://github.com/mdn/browser-compat-data/) depend on and complement each other. This document describes the workflows which synchronize the two projects.
+
+## BCD keys in `compat_features`
+
+In web-feature YAML files, the `compat_features` field consists of @mdn/browser-compat-data (BCD) entry keys (e.g., `css.properties.background-color`) that make up this feature.
+
+If `compat_features` is not set in `<feature-id>.yml`, the `dist` script will populate `compat_features` in `<feature-id>.yml.dist` with BCD entry keys tagged with `web-features:<feature-id>` in BCD, if any exist.
+
+If a `compat_features` list is in `<feature-id>.yml`, it takes precedence over BCD tags.
+
+## Tagging keys in BCD directly
+
+In this workflow, new keys in BCD are tagged from inception. This is more likely for small, incremental changes to existing features.
+
+For example, a new key appears in BCD (e.g., via a [Collector](https://github.com/openwebdocs/mdn-bcd-collector) PR). The author of that PR knew which feature the key belongs to (e.g., due to a key being renamed) and tags the key with the correct `web-features:*` tag. A few days later, that data is released by BCD. That triggers the next `@mdn/browser-compat-data` Dependabot upgrade PR, where we regenerate web-features data based on the updated tag and remove the `compat_features` list from the YAML file (if the list and tagged keys match).
```

Comment:
I would suggest re-formatting this as a list of steps with a bullet point list. Also, can you clarify "the author of that PR" please? That PR is only introduced in parenthesis before, and only by saying it's a collector PR. Might be good to clarify each step, what's the collector, why does there need to be a PR author if the collector is automated, etc.